### ISSUE TITLE
There is a question of re-training

### ISSUE THREAD

**Issue by ABexit** [c4dm/dcase-few-shot-bioacoustic#17]
Hi, there is a problem that after retraining with default setting, the F-measure is close to zero in validation set. Meanwhile, the training loss nonconvergence. Whether something wrong with the training set? Thanks for you reply.

**Comment by shubhr** [comment]
Hi.  I have updated the config.yaml and the main.py. Can you check if you are facing the same issues in training.

**Comment by ABexit** [comment]
> Hi. I have updated the config.yaml and the main.py. Can you check if you are facing the same issues in training.

Sorry to reply that after updating the config.yaml and main.py, the Training and Val accuracy still  fluctuates around 30%.

**Comment by haoheliu** [comment]
Me too... It seems so hard to reproduce the result.

**Comment by shubhr** [comment]
Hi, my apologies for that. Are you still receiving a  close to zero score on the validation set?

**Comment by shubhr** [comment]
Would it be possible for you to join a video call . I think I can look at the issue quicker than going back and forward on the message.  Let me know if we can set up a call either tomorrow or Monday first half to resolve this issue.

**Comment by haoheliu** [comment]
> Would it be possible for you to join a video call . I think I can look at the issue quicker than going back and forward on the message. Let me know if we can set up a call either tomorrow or Monday first half to resolve this issue.

Thanks for your help. How about we set up a meeting tomorrow. I'm in the UK time zone. Please set up a meeting at your convenience.

**Comment by shubhr** [comment]
> Thanks for your help. How about we set up a meeting tomorrow. I'm in the UK time zone. Please set up a meeting at your convenience.

Thank you for your reply. Would 12 PM tomorrow be ok? In case yes, then please join  https://meet.google.com/stk-icfb-csp

**Comment by haoheliu** [comment]
> 

Thanks! But in which time zone?

**Comment by shubhr** [comment]
UK time zone

**Comment by liushenme** [comment]
Hi @haoheliu @ABexit, I got the same problem that it is hard to get the reported results when retraining the deep learning based-baseline. Did you solve it?

**Comment by ABexit** [comment]
> 



> Hi @haoheliu @ABexit, I got the same problem that it is hard to get the reported results when retraining the deep learning based-baseline. Did you solve it?

I still haven't solve it. But I found that using the 2021 training set can get higher val results than baseline. Maybe you can have a try.

**Comment by liushenme** [comment]
> > 
> 
> > Hi @haoheliu @ABexit, I got the same problem that it is hard to get the reported results when retraining the deep learning based-baseline. Did you solve it?
> 
> I still haven't solve it. But I found that using the 2021 training set can get higher val results than baseline. Maybe you can have a try.

Haha, thanks for your reply. Maybe it is the problem about the 2022 training set.

**Comment by haoheliu** [comment]
> Hi @haoheliu @ABexit, I got the same problem that it is hard to get the reported results when retraining the deep learning based-baseline. Did you solve it?

Me too. I haven't solved it currently. But I found using the code and dataset from the last year can work well.

**Comment by shubhr** [comment]
Hi all, I am trying to resolve the problem but still cant find out why it is happening. On google colab, if you unzip the data to workspace and then run the code, its giving an f-score of 1.815 but if the same data is uploaded to google drive, the f-score after 2 epochs is 12.58. Its weird.. still not able to pin point the actual problem.

**Comment by shubhr** [comment]
Will post both the notebooks today.

**Comment by liushenme** [comment]
> Will post both the notebooks today.

Thanks for your reply. I hope to solve this problem soon.

**Comment by haoheliu** [comment]
Do we have an update on this issue?

**Comment by Noumanijaz744** [comment]
> Do we have an update on this issue?

hi dear i have trainig and evaluation through config.yaml set.eval:true please if you can guide me to reproduce the evaluation metrics and accuracy of the model ?
